lib/event.rb

```diff
@@ -0,0 +1,54 @@
+class Event
+  attr_reader :name,
+              :ages
+  def initialize(name, ages)
+    @name = name
+    @ages = ages
+  end
+
+  def min_age
+    min = ages[0]
+    @ages.each do |num|
+      if num < min
+        min = num
+      end
+    end
+    min
+  end
+
+  def max_age
+    max = ages[0]
+    @ages.each do |num|
+      if num > max
+        max = num
+      end
+    end
+    max
+  end
```

Comment:
Great work! There are methods called `min` and `max` that will do this for you, but I think it's valuable to do it this way. This is how it actually works under the hood, so it's great that you got that insight.